### ISSUE TITLE
Remove warning on run `serve` with one worker

### ISSUE THREAD

**Issue by vjik** [yiisoft/yii-console#212]
Currently, `-w1` show warning "number of workers must be larger than 1"

**Comment by samdark** [comment]
Yes. One worker is alright.

**Comment by klsoft-web** [comment]
> Currently, `-w1` show warning "number of workers must be larger than 1"

Yes, it is shown, but it is not related to the `yiisoft/yii-console `package, since the warning is [produced by the PHP CLI server](https://github.com/php/php-src/blob/100e8f353ea8ef841339ba4c74a0966a7eea5157/sapi/cli/php_cli_server.c#L2593).
Therefore, the issue should be closed.

**Comment by vjik** [comment]
> Yes, it is shown, but it is not related to the `yiisoft/yii-console `package, since the warning is [produced by the PHP CLI server](https://github.com/php/php-src/blob/100e8f353ea8ef841339ba4c74a0966a7eea5157/sapi/cli/php_cli_server.c#L2593). Therefore, the issue should be closed.

When workers count is `1` we should don't pass `PHP_CLI_SERVER_WORKERS` argument. It runs one worker without warning. Right?

**Comment by samdark** [comment]
Yes, correct.

**Comment by vjik** [comment]
Done by #244
